ironfish/src/account/accounts.ts

```diff
@@ -32,6 +32,8 @@ export class Accounts {
     [account: Account | null, oldAccount: Account | null]
   >()
 
+  readonly onAccountCreated = new Event<[account: Account, imported: boolean]>()
```

Comment:
if something is imported, we're not creating an account, we're importing it. I think the bool parameter and the name of this are mutually exclusive.

Reply:
Sure, I can replace this with `onAccountImported` since we don't really care about account creation in this indexer.